sql/src/test/java/io/crate/metadata/SchemasITest.java

```diff
@@ -113,17 +111,17 @@ public void testAliasPartitions() throws Exception {
         execute("create table transformer (model string, good boolean, actor object)");
         IndicesAliasesRequest request = new IndicesAliasesRequest();
         request.addAliasAction(
-            IndicesAliasesRequest.AliasActions.add().alias("entsafter").index("terminator"));
+            IndicesAliasesRequest.AliasActions.add().alias(getFqn("entsafter")).index(getFqn("terminator")));
         request.addAliasAction(
-            IndicesAliasesRequest.AliasActions.add().alias("entsafter").index("transformer"));
+            IndicesAliasesRequest.AliasActions.add().alias(getFqn("entsafter")).index(getFqn("transformer")));
```

Comment:
Wouldn't this fail now if we use the `doc` schema because the schema is then omitted in the index name?